src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.test.ts

```diff
@@ -1,46 +1,34 @@
-import { bchChainId, CHAIN_NAMESPACE, dogeChainId, ethChainId } from '@shapeshiftoss/caip'
+import { btcChainId, dogeChainId, ethChainId } from '@shapeshiftoss/caip'
 import { describe, expect, it } from 'vitest'
 
+import { UTXO_MAXIMUM_BYTES_LENGTH } from '../constants'
 import { addAggregatorAndDestinationToMemo } from './addAggregatorAndDestinationToMemo'
 import { MEMO_PART_DELIMITER } from './constants'
 
 const RECEIVE_ADDRESS = '0x32DBc9Cf9E8FbCebE1e0a2ecF05Ed86Ca3096Cb6'
 const AGGREGATOR_ADDRESS = '0xd31f7e39afECEc4855fecc51b693F9A0Cec49fd2'
-const FINAL_ASSET_ADDRESS = '0x8a65ac0E23F31979db06Ec62Af62b132a6dF4741'
 const AGGREGATOR_TWO_LAST_CHARS = 'd2'
```

Comment:
And same for this guy, expected terminology and move me inside the describe block for clear separation 
```suggestion
const EXPECTED_AGGREGATOR_TWO_LAST_CHARS = 'd2'
```

Reply:
Done in 03978b18e16